### ISSUE TITLE
MediaInsight can't be converted to Json

### ISSUE THREAD

**Issue by cosinus90** [mgp25/Instagram-API#3033]
# Bug Report

### Description

I get the following error
`Unable to convert data to array: Bad JSON property definitions in "InstagramAPI\\Response\\Model\\MediaInsights": ([\'Unexpected non-array value for array-property "reach_count" at array-depth 0 of 1.\'`
trying to execute
`$this->business->getMediaInsights($id)->asJson()`

It worked fine with earlier version of the API. But currently reach_count's type is string[]
https://github.com/mgp25/Instagram-API/blob/master/src/Response/Model/MediaInsights.php

in the same time getMediaInsights returns integer reach_count:
````
GET:  https://i.instagram.com/api/v1/insights/media_organic_insights/2169618742145699409/?ig_sig_key_version=4
→ 0B
← 200    189B
RESPONSE: {"media_organic_insights": {"like_count": 1, "comment_count": 0, "save_count": 0, "impression_count": 4, "reach_count": 3, "engagement_count": 1, "avg_engagement_count": 0}, "status": "ok"}

````

I've rolled back my code to int type and it works smoothly. I guess there was a reason for changing it to string[] by this commit
https://github.com/mgp25/Instagram-API/commit/c6ebbed2a8a5c5b663b99446b45a98f55d472c6a#diff-a5f6597b71cbe4ed5227e4ea18ff6b83
but looks like Instagram varies response.

**Comment by JRoy** [comment]
Could you please share the response which is giving this error?

**Comment by cosinus90** [comment]
It was posted before, repeating
````
GET:  https://i.instagram.com/api/v1/insights/media_organic_insights/2169618742145699409/?ig_sig_key_version=4
→ 0B
← 200    189B
RESPONSE: {"media_organic_insights": {"like_count": 1, "comment_count": 0, "save_count": 0, "impression_count": 4, "reach_count": 3, "engagement_count": 1, "avg_engagement_count": 0}, "status": "ok"}
````
or may be do you mean something else?

**Comment by JRoy** [comment]
Should be fixed in 359d840dbc0ffd9da49c6a92ef7b9aa77970facc

Regards.
